docs/installingLocal.md

```diff
@@ -83,3 +83,43 @@ cd /PrairieLearn
 ./docker/test_js.sh
 ./docker/test_python.sh
 ```
+
+### Connecting to an existing Docker container
+
+The previous shells were launched in their own containers. If you want to open a shell in a Docker container that is *already running*, you can find the container's name and connect to it.
+
+1. Find the name of your running PrairieLearn container by running
+
+```sh
+docker ps
+```
+
+which will output multiple columns of information about your running container(s). Look for the `prairielearn/prairielearn` image and copy its corresponding name. For example, the name of the PrairieLearn container in this `docker ps` output is `upbeat_roentgen`:
+
+```
+CONTAINER ID  IMAGE                      COMMAND              CREATED      STATUS      PORTS                   NAMES
+4be50d160d49  catracker-devcontainer     "sleep infinity"     5 hours ago  Up 5 hours                          determined_cori
+c681714bea9a  catracker-postgres         "docker-entrypoin…"  5 hours ago  Up 5 hours  5432/tcp                tender_wu
+e0f522f41ea4  prairielearn/prairielearn  "/bin/sh -c /Prai…"  2 hours ago  Up 2 hours  0.0.0.0:3000->3000/tcp  upbeat_roentgen
+```
```

Comment:
Too busy? Maybe most users will only see 1 line.

Reply:
I agree it seems unnecessarily busy. I think it's probably best to go with the minimal example that most users will see (single PL container). Users who are running multiple containers can presumably figure things out 😄